rpc/handlers.go

```diff
@@ -229,8 +230,9 @@ func buildJsonRPCRelayPayload(body *RelayRequest) *coreTypes.Relay {
 		payload.JsonRpcPayload.Id = []byte(*body.Payload.Id)
 	}
 
-	if body.Payload.Parameters != nil {
-		payload.JsonRpcPayload.Parameters = *body.Payload.Parameters
+	if body.Payload.Params != nil {
+		// ADDPR: Need a decision and implementation on Params field and conversion from rpc to proto
```

Comment:
1. Does this need to be done in this PR?
2. Can you provide more details (potentially with a code / structure example)?

Reply:
Appreciate all the details.

> may need some careful consideration

Given that this is such a big question that requires a discussion, it seems like a task that is outside the scope of this "mini PR" to get some functional demo merged to main. 

Wdyt?

Reply:
Sure, I'll update the item to `DISCUSS`.